src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainView.scss

```diff
@@ -6,5 +8,120 @@
 }
 
 .speciesLabelBlock {
-  padding: 0 75px 0 60px;
+  padding: 0 75px 0 45px;
+}
+
+.statsWrapper { 
+    padding: 5px 20px 40px 120px;
+    max-width: 1310px;
+}
+
+.exampleLinkText{
+    cursor: pointer;
+    color:$blue;
 }
+
+
+.pointerBox {
+    padding: 6px 12px;
+    background: $black;
+    color: white;
+  
+    p {
+      max-width: 200px;
+    }
+  }
+  
+  .pointerBoxPointer {
+    fill: $black;
+  }
+  
+
+.collapsedContent{
+  display: grid;
+  padding:6px 20px;
+  grid-template-columns: [title] 120px [primary] 250px [secondary] auto [example-link] 150px;
+  align-items: center;
+
+  .title{
+      grid-column: title;
+      color: $medium-dark-grey;
+  }
+
+  .primary{
+      grid-column: primary;
+
+      .primaryValue{
+          font-size: 22px;
+      }
+
+      .primaryUnit{
+          color: #6f8190;
+          font-size: 13px;
+          padding-left: 5px;
+      }
+  }
+
+  .secondary{
+      grid-column: secondary;
+
+      .secondaryValue{
+          font-size: 22px;
+      }
+
+      .secondaryUnit{
+          color: #6f8190;
```

Comment:
this color is defined in our design system as a variable

Reply:
Updated it